packages/client-core/src/admin/components/locations/AddEditLocationModal.tsx

```diff
@@ -317,56 +318,11 @@ export default function AddEditLocationModal(props: {
               onChange={screenSharingEnabled.set}
               disabled={isLoading}
             />
-            <div>{t('editor:properties.sceneSettings.lbl-thumbnail')}</div>
-            <div className="flex flex-col ">
-              <div className="flex flex-row justify-around">
-                <div>{'Current Thumbnail'}</div>
-                <div>{'Previous Thumbnail'}</div>
-              </div>
-              <div className="flex flex-row justify-evenly">
-                <ImageLink src={sceneThumbnailState.thumbnailURL.value ?? ''} variant="md" />
-                <ImageLink src={sceneThumbnailState.oldThumbnailURL.value ?? ''} variant="md" />
-              </div>
-              <div className="flex flex-row gap-2 ">
-                <Button onClick={SceneThumbnailState.createThumbnail} className="w-full">
-                  {t('editor:properties.sceneSettings.generate')}
-                </Button>
-                <Button
-                  onClick={() => {
-                    SceneThumbnailState.uploadThumbnail(sceneSettingsEntities)
-                  }}
-                  disabled={!sceneThumbnailState.thumbnail.value}
-                  className="w-full"
-                >
-                  {t('editor:properties.sceneSettings.save')}
-                </Button>
-              </div>
-            </div>
-            <div>{t('editor:properties.sceneSettings.lbl-loading')}</div>
-            <div className="flex flex-col">
-              <div className="flex flex-row justify-around">
-                <div>{'Current Loading Screen'}</div>
-                <div>{'Previous Loading Screen'}</div>
-              </div>
-              <div className="flex flex-row justify-evenly ">
-                <ImageLink src={sceneThumbnailState.loadingScreenURL.value ?? ''} variant="md" />
-                <ImageLink src={sceneThumbnailState.oldLoadingScreenURL.value ?? ''} variant="md" />
-              </div>
-              <div className="flex flex-row gap-2">
-                <Button onClick={SceneThumbnailState.createLoadingScreen} className="w-full">
-                  {t('editor:properties.sceneSettings.generate')}
-                </Button>
-                <Button
-                  onClick={() => {
-                    SceneThumbnailState.uploadLoadingScreen(sceneSettingsEntities)
-                  }}
-                  disabled={!sceneThumbnailState.loadingScreenImageData.value}
-                  className="w-full"
-                >
-                  {t('editor:properties.sceneSettings.save')}
-                </Button>
-              </div>
-            </div>
+            {engineState.viewerEntity.value !== UndefinedEntity && (
```

Comment:
rather than using engineState here, we should pass in a boolean argument to the react component that specifies whether or not we should render this. we want to avoid using any ecs/spatial/engine imports at all

Reply:
gotcha